app/routes/languages.rb

```diff
@@ -70,6 +70,11 @@ def topic_not_found(topic)
         status 404
         erb :"languages/not_found", locals: { track_id: topic }
       end
+
+      def active(topic, selected_topic)
+        'active' if topic == selected_topic
+      end
```

Comment:
You previously asked to move it out of the erb file. Do you mean to define the method in the erb file?